src/url-replacements.js

```diff
@@ -82,7 +83,16 @@ class UrlReplacements {
     // single page view. It should have sufficient entropy to be unique for
     // all the page views a single user is making at a time.
     this.set_('PAGE_VIEW_ID', () => {
-      documentInfoFor(this.win_).pageViewId;
+      return documentInfoFor(this.win_).pageViewId;
+    });
+
+    this.set_('CLIENT_ID', (data, name) => {
+      return cidFor(this.win_).then(cid => {
```

Comment:
Yeah, I've thought about that as well. We definitely don't want CID in core, though.